R/render_comms_doc.R

```diff
@@ -24,11 +26,10 @@ render_comms_doc = function(info, outformat = "", template_url = "https://raw.gi
 
   # download the correct template and give it the name of the slug plus the template name
 
-  doc_loc <- paste0("files/", info$slug, "/")
   doc_name <- paste0(info$slug, "_", doctype, "_doc")
 
   comms_templ <- paste0(template_url, doctype, loctype, "_doc.Rmd") # URL to the Rmd template
-  download_name <- paste0(doc_loc, doc_name, ".Rmd")
+  download_name <- paste0(doc_name, ".Rmd")
```

Comment:
is this based on the other PR? This seems like something not to mess with, but maybe I was the one who did this..

Reply:
No, I did this because it created an error. The name of the downloaded file was used later in `rmarkdown::render` as the input document, but it's not possible (as far as I could deduce) to have this input document in a subdirectory; it told me that the subdirectory didn't exist (even if it did).

I ended up removing the subdirectory and this fixed the function. We need to look at where files are downloaded to and rendered to anyway in #27, so I figured I'd kick that can down the road...